themes/kube/static/css/kube.css

```diff
@@ -1012,24 +1015,30 @@ button:hover,
 .button.small {
   font-size: 13px;
   min-height: 36px;
-  padding: 6px 20px;
+  line-height: 36px;
+  vertical-align: middle;
+  padding: 0 20px;
   border-radius: 3px; }
 
 .button.big {
   font-size: 17px;
   min-height: 48px;
-  padding: 13px 24px;
+  line-height: 48px;
+  vertical-align: middle;
+  padding: 24px;
```

Comment:
Maybe we do not need to change the big button, WDYT

Reply:
Oh sorry for that, and thanks for telling!
The former `button small` CSS used `padding` feature and managed to make text in the vertical middle, same as `button large` and `button big`, but that's not a good way as you can see in my comment. So I change those CSS and use `line-height` & `vertical-align` feature. But I forgot to make `padding-top` & `padding-small` as `0`.
Now I fixed this bug.